new-frontiers-14/frontier-station-14
# Add GPS app to pirate PDAs

## About the PR


Added the AstroNav program on pirate PDAs !

## Why / Balance

Pirate players tend to get lost in space with no ability to find where they are!

## How to test

- Spawn as a pirate/pirate first mate/pirate captain
- Take the PDA in hand to check if the coordinates are shown on the held item interface
- Open the Programs tab to check if the AstroNav program is installed

## Media

![piratepda1](https://github.com/user-attachments/assets/61ede85f-d60e-4b83-8b60-84bd631075b3)


## Requirements



## Breaking changes


**Changelog**
:cl:
- tweak: Pirate PDAs now have the AstroNav program
